src/thor/timedistancematrix.cc

```diff
@@ -121,7 +121,7 @@ void TimeDistanceMatrix::ExpandForward(GraphReader& graphreader,
     // Add to the adjacency list and edge labels.
```

Comment:
this algorithm is the matrix equivalent of timedep_forward/rev for completeness we should copy paste the same changes we did elsewhere to here. we can add matrix unit testing to confirm it later on.